README.md

```diff
@@ -1,96 +1,117 @@
 2. Raise a PR with an implementation sketch.
 3. Once you have some consensus, ask an admin to create a feature branch against which PRs can be raised.
 4. When the design has stabilized and is functional enough to be useful, raise a PR to merge the new module to master.
 
-To propose a new feature for an existing module:
+### How to propose a new feature for an existing module
+
 1. Raise an issue with the [enhancement] tag.
 2. Raise a PR with your implementation, and discuss the implementation there.
 3. Once there is a consensus that the new feature is desirable and the design is suitable, it can be merged.
 
-To fix a bug, or make smaller improvements:
-1. Raise a PR with your change. Be sure to add test coverage for whatever changes you are making.
+### How to fix a bug, or make smaller improvements
 
-Questions about how to use Swift Numerics modules, or issues that are not clearly bugs can be discussed in the ["Swift Numerics" section of the Swift forums.](https://forums.swift.org/c/related-projects/swift-numerics)
+1. Raise a PR with your change. 
+2. Make sure to add test coverage for whatever changes you are making.
+
+### Forums
+
+Questions about how to use Swift Numerics modules, or issues that are not clearly bugs can be discussed in the ["Swift Numerics" section of the Swift forums](https://forums.swift.org/c/related-projects/swift-numerics).
 
 ## Modules
-1. [RealModule](Sources/RealModule/README.md)
-2. [ComplexModule](Sources/ComplexModule/README.md)
+
+1. About [`RealModule`](Sources/RealModule/README.md)
+2. About [`ComplexModule`](Sources/ComplexModule/README.md)
 
 ## Future expansion
-1. [Approximate Equality](https://github.com/apple/swift-numerics/issues/3)
-2. [Large Fixed-Width Integers](https://github.com/apple/swift-numerics/issues/4)
-3. [Arbitrary-Precision Integers](https://github.com/apple/swift-numerics/issues/5)
-4. [Shaped Arrays](https://github.com/apple/swift-numerics/issues/6)
-5. [Decimal Floating-point](https://github.com/apple/swift-numerics/issues/7)
+
+1. [Large Fixed-Width Integers](https://github.com/apple/swift-numerics/issues/4)
+2. [Arbitrary-Precision Integers](https://github.com/apple/swift-numerics/issues/5)
+3. [Shaped Arrays](https://github.com/apple/swift-numerics/issues/6)
+4. [Decimal Floating-point](https://github.com/apple/swift-numerics/issues/7)
 
 ## Notes
-¹ Swift is currently unable to use the fully-qualified name for types when a type and module have the same name (discussion here: https://forums.swift.org/t/pitch-fully-qualified-name-syntax/28482).
-This would prevent users of Swift Numerics who don't need generic types from doing things like:
+
+¹ Swift is currently unable to use the fully-qualified name for types when a type and module have the same name (discussion here: https://forums.swift.org/t/pitch-fully-qualified-name-syntax/28482). This would prevent users of Swift Numerics who don't need generic types from doing things, such as:
```

Comment:
```suggestion
¹ Swift is currently unable to use the fully-qualified name for types when a type and module have the same name (discussion here: https://forums.swift.org/t/pitch-fully-qualified-name-syntax/28482).
This would prevent users of Swift Numerics who don't need generic types from doing things, such as:
```

Reply:
👍👍